.github/workflows/R-CMD-check.yaml

```diff
@@ -45,7 +45,9 @@ jobs:
         with:
           extra-packages: any::rcmdcheck
           needs: check
-
+      - name: Run redis
+        if: ${{ matrix.config.os != 'macos-latest' }}
```

Comment:
We should just drop non-Linux builds from the matrix, we will never run the server on these anyway

Reply:
Fine by me!